### ISSUE TITLE
$\sqrt{x}$ can't be rendered properly after enabled this extension

### ISSUE THREAD

**Issue by m1stm4o** [phoihos/vscode-markdown-markmap#1]
Hi, I installed the extension "Markdown All in One" before, which can render latex, this function is OK. Recently I installed this extension "vscode-markdown-markmap", while after that some latex can't work properly in markdown.

The picture shows the problem.
If there's no character behind "$\sqrt{x}$", "$\sqrt{x}$" can't be rendered properly, which is not correct.
<img width="262" alt="pic" src="https://user-images.githubusercontent.com/16450904/98468673-83281880-2216-11eb-8fbc-72db2b9046d4.png">

If you want to reproduce, just install the extensions "Markdown All in One" and "vscode-markdown-markmap", and copy the following code to a markdown file
```
$\sqrt{x}$

$\sqrt{x}$a

$\sqrt{x}$  
n

${\sqrt{x}}$

$\sqrt{{x}}$
```

It would be great If you can do some check about this. Thank you!:smile:

**Comment by phoihos** [comment]
@m1stm4o 
Thanks for your reporting. I'll get it investigated.

**Comment by phoihos** [comment]
This happens because the `markdown-it-attr` module in "vscode-markdown-markmap" was conflicted with `{ }` curly brackets to the `markdown-it-katex` module in "Markdown All in One" extension.

I'll add some options to solve this issue soon.
Thank you again for your reporting.

**Comment by m1stm4o** [comment]
Ah, I see, looking forward to your update.

**Comment by phoihos** [comment]
@m1stm4o 
This issues fixed in `vscode-markdown-markmap` v1.1.0.
Now, you can use `{ }` curly brackets to add attributes with or without the `markdown-it-attr` module.

**Comment by m1stm4o** [comment]
@phoihos Thanks.
